denops/@ddc-filters/converter_fuzzy.ts

```diff
@@ -20,17 +20,22 @@ export class Filter extends BaseFilter<Params> {
         const match = fuzzy.findBestMatch(
           normalize(args.completeStr),
           normalize(item.word),
-        )
-        return {
-          ...item,
-          highlights: match?.pos.map((col) => ({
-            col: col,
-            type: "abbr",
-            name: "ddc_fuzzy_matched_character",
-            "hl_group": args.filterParams.hlGroup,
-            width: 1,
-          })),
-        };
+        );
+        return match
```

Comment:
Instead of `... ? ... : ...`, could you write as follows:

```
if (!match) {
  return item
}
return {
  ...
}
```

Reply:
Changed it.